packages/config/tests/api/tests.js

```diff
@@ -136,93 +129,91 @@ test('--account-id in offline and buildbot mode', async (t) => {
 })
 
 test('NETLIFY_SITE_ID environment variable', async (t) => {
-  const output = await new Fixture('./fixtures/empty')
-    .withEnv({ NETLIFY_SITE_ID: 'test' })
-    .runWithConfig([FETCH_INTEGRATIONS_EMPTY_RESPONSE])
```

Comment:
What I find strange is that `runWithConfig` doesn't seem to accept arguments. No action needed, just thinking out loud.

Reply:
I thought this too, but then found that what this *actually* means is "run this test with the `@netlify/config` package", not "run something with the config we're passing in here". The parameter it accepts is an array of our... uh.. _bespoke_ mocked out http responses.